src/element.ts

```diff
@@ -351,12 +364,15 @@ export abstract class Element {
    * Each Element subclass may specify its own default by overriding the static `TEXT_FONT` property.
    */
   setFont(font?: string | FontInfo, size?: string | number, weight?: string | number, style?: string): this {
-    // Allow subclasses to override `TEXT_FONT`.
-    const defaultTextFont: Required<FontInfo> = (<typeof Element>this.constructor).TEXT_FONT;
+    const defaultTextFont: Required<FontInfo> = {
+      family: Tables.lookupMetric(`${this.#attrs.type}.fontFamily`),
+      size: Tables.lookupMetric(`${this.#attrs.type}.fontSize`),
+      weight: Tables.lookupMetric(`${this.#attrs.type}.fontWeight`),
+      style: Tables.lookupMetric(`${this.#attrs.type}.fontStyle`),
+    };
```

Comment:
It would be nice if this could be handled on a single line. Something like:

```js
this.textFont = Tables.lookupMetric(this.#attrs.type.font);
```

Reply:
done